src/main/java/com/ludogorieSoft/villagelifefrontend/enums/Distance.java

```diff
@@ -0,0 +1,16 @@
+package com.ludogoriesoft.villagelifefrontend.enums;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public enum Distance {
+    IN_THE_VILLAGE("в селото"),
+    ON_10_KM("на 10 км"),
+    ON_11_TO_30KM("на 11-30 км"),
+    ON_31_TO_50_KM("на 31-50 км"),
```

Comment:
the client said that we dont need higher distance tha 25km for the points of interests around the village